pypeit/reduce.py

```diff
@@ -664,6 +671,17 @@ def global_skysub(self, skymask=None, update_crmask=True, trim_edg=(3,3),
         # Mask objects using the skymask? If skymask has been set by objfinding, and masking is requested, then do so
         skymask_now = skymask if (skymask is not None) else np.ones_like(self.sciImg.image, dtype=bool)
 
+        # Allow for previous sky to better estimate ivar
+        if previous_sky is None:
+            ivar = self.sciImg.ivar 
+        else:
+            # Estimate the variance using the input sky model
+            var = procimg.variance_model(self.sciImg.base_var, 
```

Comment:
Note this implementation breaks IR reductions since there we fit residuals and never have any choice but to use the image as the variance, since it makes no sense to use the sky_residuals as the noise model. Let's talk.

Reply:
Good point. 

I have reverted it to behave as before for `ir_redux=True`